plugins/tracker-resources/src/components/issues/IssueNotification.svelte

```diff
@@ -73,6 +67,6 @@
 
   <svelte:fragment slot="buttons">
     <Button label={tracker.string.ViewIssue} on:click={handleIssueOpened} />
-    <Button icon={view.icon.CopyLink} label={tracker.string.CopyIssueUrl} on:click={handleCopyUrl} />
+    <Button icon={view.icon.CopyLink} label={tracker.string.CopyIssueUrl} on:click={() => copyTextToClipboard(params?.issueUrl)} />
```

Comment:
It will not pass validation, since copyTextToClipboard accept string, not `string|undefined`